log/log.go

```diff
@@ -222,11 +235,24 @@ func Debug(ctx context.Context, fields map[string]interface{}, format string, ar
 
 // extractCallerDetails gets information about the file, line and function that
 // called a certain logging method such as Error, Info, Debug, Warn and Panic.
-func extractCallerDetails() (string, int, string, error) {
+func extractCallerDetails() (string, int, string, string, error) {
```

Comment:
IMHO you should either give the multi return values a name in the signature of this function or return a struct.